src/util/system.h

```diff
@@ -82,6 +84,16 @@ fs::path GetSpecialFolderPath(int nFolder, bool fCreate = true);
 #if HAVE_SYSTEM
 void runCommand(const std::string& strCommand);
 #endif
+#ifdef HAVE_BOOST_PROCESS
+/**
+ * Execute a command with returns JSON, and parse the result.
```

Comment:
Typo, witch.

Also, could say what happens if the output can't be parsed.

Reply:
> Typo, witch.

Which, seems like like a typo witch hunt.

Reply:
Ah! I wanted to type "with".

Reply:
A typo seems remain :)
```suggestion
 * Execute a command which returns JSON, and parse the result.
```